src/app/inventory/store/d2-item-factory.service.ts

```diff
@@ -1209,6 +1199,33 @@ function buildSocket(
   };
 }
 
+function buildForsakenMasterworkInfo(createdItem: D2Item, defs: D2ManifestDefinitions) {
+  const masterworkSocket = createdItem.sockets!.sockets.find((socket) => {
+    return !!(
+      socket.plug && socket.plug.plugItem.plug.plugCategoryIdentifier.includes('masterworks.stat')
+    );
+  });
+  if (masterworkSocket && masterworkSocket.plug) {
+    // createdItem.masterwork = true;
```

Comment:
Do we want `is:masterwork` to show items that have the masterwork slot? My first thought was no, but I wasn't sure how we'd want to do filters for that.

Reply:
No. Because every Y2 item is at least masterwork 1.